ajax/libs/leaflet-gpx/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "leaflet-gpx",
+  "filename": "gpx.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mpetazzoni/leaflet-gpx.git",
+    "basePath": "",
+    "files": [
+      "gpx*"
```

Comment:
Because I think whole the ```gpx...``` and ```pin-...``` files should be added.
So I use that form.
Thanks.